lib/atom/mob/index.js

```diff
@@ -1,22 +1,24 @@
 'use strict';
 
 const {chain_func, is_atom, to_chat, has_component} = require('../utils.js');
-const _observers = Symbol('_observers');
-const _viewers = Symbol('_viewers');
-const _viewing = Symbol('_viewing');
-const _key = Symbol('_key');
-const _client = Symbol('_client');
-const _server_to_net = Symbol('_server_to_net');
-const _visible_tiles = Symbol('_visible_tiles');
-const _screen_set = Symbol('_screen_set');
-const _update_var = Symbol('_update_var');
-const _add_viewing = Symbol('_add_viewing');
-const _remove_viewing = Symbol('_remove_viewing');
-const _common_tiles_count = Symbol('_common_tiles_count');
-const _panel_map = Symbol('_panel_map');
-const _visgroups = Symbol('_visgroups');
-const _eye_to_eyeid = Symbol('_eye_to_eyeid');
-module.exports._symbols = {_observers, _client, _viewers, _viewing, _visible_tiles, _server_to_net, _add_viewing, _remove_viewing, _update_var, _common_tiles_count, _visgroups, _key};
+const _symbols = require('./symbols.js');
+
+const _observers = _symbols._observers;
```

Comment:
can't you do `const {_observers, _viewers, _viewing, etc} = require('./symbols.js')`  and have it all be one line?

Reply:
Oh, sure, just wanted to make as few changes as possible

Reply:
Well, two lines since there's also `module.exports._symbols`